codeserver/ubi9-python-3.11/Dockerfile.cpu

```diff
@@ -136,7 +136,7 @@ COPY ${CODESERVER_SOURCE_CODE}/Pipfile.lock ./
 
 # Install packages and cleanup
 RUN echo "Installing softwares and packages" && \
-    micropipenv install && \
+    micropipenv requirements | uv pip install --strict --no-deps --no-cache --no-config --no-progress --require-hashes --index-strategy unsafe-best-match --requirements - && \
```

Comment:
One question - why `--no-deps`?

Reply:
Because the `requirements.txt` is already fully resolved including with transitive dependencies.

Reply:
ah, so this is just to avoid another resolving operation... understand